jmeter-service/testrunner.go

```diff
@@ -158,14 +161,14 @@ func (tr *TestRunner) runHealthCheck(testInfo TestInfo, testStartedAt time.Time,
 		if err := tr.sendErroredTestsFinishedEvent(testInfo, testStartedAt, msg); err != nil {
 			logger.Errorf(errMsgSendFinishedEvent, err, testInfo)
 		}
-		return nil
+		return err
 	}
 	if !res {
 		msg := fmt.Sprintf("Tests for %s with status = %s. %v", TestStrategy_HealthCheck, strconv.FormatBool(res), testInfo)
 		if err := tr.sendTestsFinishedEvent(testInfo, testStartedAt, msg, keptnv2.ResultFailed); err != nil {
 			logger.Errorf(errMsgSendFinishedEvent, err, testInfo)
 		}
-		return nil
+		return errors.New(msg)
```

Comment:
I think we can keep `return nil` here, as a failed jmeter test is not necessarily an error in the execution of the `test` task